.github/workflows/acceptance.yml

```diff
@@ -0,0 +1,198 @@
+name: acceptance
+on: pull_request
+
+jobs:
+    percynonce:
+      name: Generate Percy nonce
+      runs-on: ubuntu-latest
+      steps:
+        - name: Generate Percy nonce
+          if: always()
+          run: |
+            echo ${{ github.run_id }}.$(($(date +%s))) > nonce.txt
+
+        - name: Upload nonce as artifact
+          if: always()
+          uses: actions/upload-artifact@v1
+          with:
+            name: percy
+            path: nonce.txt
+
+    acceptance:
+      needs: [percynonce]
+      runs-on: ubuntu-latest
+      strategy:
+        matrix:
+          instance: [0, 1, 2]
+
+      env:
+        PIP_DISABLE_PIP_VERSION_CHECK: on
+        PIP_QUIET: 1
+
+        SENTRY_LIGHT_BUILD: 1
+        SENTRY_SKIP_BACKEND_VALIDATION: 1
+        MIGRATIONS_TEST_MIGRATE: 0
+
+        # Use this to override the django version in the requirements file.
+        DJANGO_VERSION: ">=1.11,<1.12"
+
+        # Node configuration
+        # node's version is pinned by .nvmrc and is autodetected by `nvm install`.
+        NODE_OPTIONS: --max-old-space-size=4096
+        NODE_ENV: development
+
+        PYTEST_SENTRY_DSN: https://6fd5cfea2d4d46b182ad214ac7810508@sentry.io/2423079
+        PYTEST_ADDOPTS: "--reruns 5"
+
+        # services configuration
+        SENTRY_KAFKA_HOSTS: kafka:9093
+        SENTRY_ZOOKEEPER_HOSTS: zookeeper:2182
+        SENTRY_REDIS_HOST: redis
+        # The hostname used to communicate with the PostgreSQL service container
+        POSTGRES_HOST: postgres
+        # The default PostgreSQL port
+        POSTGRES_PORT: 5432
+
+        # Percy config
+        PERCY_TOKEN: ${{ secrets.PERCY_TOKEN }}
+        PERCY_PROJECT: ${{ secrets.PERCY_PROJECT }}
+
+        # Number of matrix instances
+        TOTAL_TEST_GROUPS: 3
+
+      services:
+        clickhouse:
+          image: yandex/clickhouse-server:19.11
+          options: --ulimit nofile=262144:262144
+          ports:
+            - 9000:9000
+
+        snuba:
+          image: getsentry/snuba
+          env:
+            SNUBA_SETTINGS: test
+            REDIS_HOST: redis
+            CLICKHOUSE_HOST: clickhouse
+            CLICKHOUSE_PORT: 9000
+          ports:
+            - 1218:1218
+
+        redis:
+          image: redis
+          ports:
+            - 6379:6379
+
+        memcached:
+          image: memcached
+          ports:
+            - 11211:11211
+
+        postgres:
+          image: postgres:10.8
```

Comment:
prod + devservices use postgres 9.6 - this should match that.

Although actually now that I'm looking at this, Travis might not be running 9.6. 

Couldn't find the default postgres version for their xenial images, but they let you pin it: https://docs.travis-ci.com/user/database-setup/#using-a-different-postgresql-version

I can test that and put on separate PR soon.

Reply:
```suggestion
          image: postgres:9.6
```

Reply:
Doesn't seem to like 9.6

Reply:
Dang, that's kind of a dealbreaker though... well, not so much for acceptance tests (i assume not that much db stuff is happening) but backend tests. Does retrying work? Maybe just random github issue.

Reply:
OK, turns out it just needs a password.

Reply:
:thonk:

Reply:
> well, not so much for acceptance tests (i assume not that much db stuff is happening)

Acceptance tests run the entire app, so db stuff definitely happens.